doc/auth/saml.md

```diff
@@ -0,0 +1,133 @@
+## Overview
+
+This guide will walk through using Pachyderm's SAML support.
+These features aren't integrated into mainline Pachyderm yet and aren't
+available in any official releases. This will describe the process of:
+
+1. Activating Pachyderm enterprise and Pachyderm auth
+1. Configuring Pachyderm's auth system to enable its SAML ACS and receive SAML
+    assertions
+1. Logging in to both the dash and CLI
+1. Enabling debug logging in case anything goes wrong
+
+## Activation
+
+When starting out, we **highly** recommend running Pachyderm in Minikube, in case
+any configuration bugs make it necessary to restart the cluster.
+
+To activate Pachyderm enterprise and Pachyderm auth:
+
+```
+pachctl enterprise activate <enterprise code>
+pachctl auth activate --initial-admin=robot:admin
+```
+
+At this point, Pachyderm is ready to authenticate & authorize users.
+
+What the `--initial-admin` flag does is this:
+1. Pachyderm requires there to be at least one cluster admin if auth is
+   activated
+1. Pachyderm's authentication is built around GitHub by default. Without this
```

Comment:
Can we make these ascending numbers for those who read markdown in text format?

Reply:
Done